cocotb/handle.py

```diff
@@ -398,17 +410,17 @@ def _getvalue(self):
                 result.append(self[x]._getvalue())
             return result
         else:
-            raise TypeError("Not permissible to get values on object %s type %s" % (self._name, type(self)))
+            raise TypeError("Not permissible to get values of object %s of type %s" % (self._name, type(self)))
 
     def setimmediatevalue(self, value):
-        raise TypeError("Not permissible to set values on object %s type %s" % (self._name, type(self)))
+        raise TypeError("Not permissible to set values on object %s of type %s" % (self._name, type(self)))
 
     def _setcachedvalue(self, value):
-        raise TypeError("Not permissible to set values on object %s type %s" % (self._name, type(self)))
+        raise TypeError("Not permissible to set values on object %s of type %s" % (self._name, type(self)))
 
     def __le__(self, value):
-        """Overload the less than or equal to operator to
-            provide an hdl-like shortcut
+        """Overload the less-than-or-equal-to operator to
+            provide an HDL-like shortcut
```

Comment:
Indent is very weird here. Docstring ought to have a one-line summary, followed by two blank lines.

Reply:
Done.